bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue

```diff
@@ -490,7 +490,7 @@ export default {
     },
     toggleDisabled () {
       const enable = (this.thing.statusInfo.statusDetail === 'DISABLED')
-      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString(), 'application/json', 'application/json').then((data) => {
+      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString(), 'text/plain', 'text/plain').then((data) => {
```

Comment:
```suggestion
      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString(), 'application/json', 'text/plain').then((data) => {
```

I believe the enable endpoint returns JSON, but consumes text.

 You could also just omit the last two parameters as they are the default types
```suggestion
      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString()).then((data) => {
```

Reply:
Thanks, done.